container/options.go

```diff
@@ -216,6 +245,23 @@ func SplitPercent(p int) SplitOption {
 	})
 }
 
+// SplitFixed sets the size as a fixed value of the available space.
+// When using SplitVertical, the provided size is applied to the new left
+// container, the new right container gets the reminder of the size.
+// When using SplitHorizontal, the provided size is applied to the new top
+// container, the new bottom container gets the reminder of the size.
+// The provided value must be a positive number in the range 0 <= cells.
+// If not provided, defaults to DefaultSplitFixed.
```

Comment:
The note about the "default" might not be correct. Since when this isn't provided, this really defaults to using SplitPercent with its default value of  DefaultSplitPercent.

Reply:
Ah, yeah that's more accurate.
Changed to
```
// If SplitFixed() is not specified, it defaults to SplitPercent() and its given value.
```